### DISCUSSION TITLE
AOMP and/or openmp-extras

### DISCUSSION THREAD

**Discussion by JP-Ellis** [rocm-arch/rocm-arch#683] [Ideas]
Firstly, I'd like to say that I am not intimately familiar with this, so apologies if there is a misunderstanding.

I was trying to update `openmp-extras` to v5.0.1 seeing as it hadn't been done yet, but ran into some difficulties.  In looking back in previous commits, I found that there used to be an `aomp-amdgpu` package which looks like it was superceded by `openmp-extras` (#463, #627).  The new `openmp-extras` package was based on instruction provided in the aomp documentation; however, in [recent version of the aomp documentation](https://github.com/ROCm-Developer-Tools/aomp/tree/aomp-dev/docs), I could not find the instructions for `openmp-extras`.

So now I am wondering whether the AOMP package should be reintroduced, and whether `openmp-extras` should be deprecated?

**Comment by tpkessler** [comment]
Hey @JP-Ellis! `aomp-amdgpu` was deprecated by AMD a while back. The aomp compiler is now only used for the development of the latest OpenMP-related LLVM changes and is not packaged as a standalone application anymore. `ompmp-extras` is the integrated version of aomp, i.e. it uses hsa-rocr and alike from the pre-installed ROCm stack instead of building its own version.

> ran into some difficulties.

You can open separate issues on this and we can work together on a fix. I'm tagging @linedot as the author of the first working PKGBUILD for openmp-extra.

**Comment by linedot** [comment]
Ok I made a pull request, please take a look: https://github.com/rocm-arch/rocm-arch/pull/685

**Comment by tpkessler** [comment]
`openmp-extras` was updated to 5.0.2
